src/main/java/org/wildfly/security/util/Alphabet.java

```diff
@@ -269,4 +269,41 @@ public int decode(final int codePoint) {
             }
         };
     }
+
+    /**
+     * The alphabet used by PicketBox project base 64 encoding.
+     * {@code 0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz./}
+     */
+    public static final Base64Alphabet PICKETBOX_BASE_64 = new Base64Alphabet(false) {
```

Comment:
The ``_BASE_64`` suffix is redundant here, as all the alphabets are base 64 alphabets.  It could just be ``PICKETBOX``, or better yet, named after the standard name of this alphabet (if there is one).